front-end/src/components/aptdetails/Features.jsx

```diff
@@ -51,10 +50,10 @@ const Features = ({ apartmentData }) => {
 
   return (
     <React.Fragment>
-      <Card sx={{ height: 350, textAlign: "center" }}>
+      <Card sx={{ height: 350, textAlign: "center", border: 0.5 }}>
         <Typography
-          className=" text-white text-center py-3 text-lg font-semibold rounded-t-lg"
-          bgcolor={"#206129"}
+          className="text-center py-3 text-lg font-semibold"
+          bgcolor={"#e3e7d3"}
         >
           Features
         </Typography>
```

Comment:
Same thing here, make sure the mt on the Add/remove features button added stays if a conflict arises near line 61